lib/cisco_node_utils/cmd_ref/snmp_server.yaml

```diff
@@ -63,7 +63,7 @@ packet_size:
     default_value: 0
   N6k: *n5k_default_packet_size
   N7k: *n5k_default_packet_size
-  N9k-F: *n5k_default_packet_size
+  N9k-F: *n3k_default_packet_size
```

Comment:
Have you tested this change on both the original N9K-Fretta images and the latest?